modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/schema/IgniteTableImpl.java

```diff
@@ -78,7 +75,7 @@
     public IgniteTableImpl(GridKernalContext ctx, TableDescriptor desc) {
         this.ctx = ctx;
         this.desc = desc;
-        statistic = new StatisticsImpl();
+        //statistic = new StatisticsImpl();
```

Comment:
fix it

Reply:
why do u resolve it ? i see no changes

Reply:
cause I did local changes and write about it. Not I pushed it.